src/database/migrations/20220411131632-create-trip-accomodation-relationship.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable require-jsdoc */
+export async function up(queryInterface, Sequelize) {
+  await queryInterface.addColumn('trips', 'accomodationId', {
+    type: Sequelize.INTEGER,
+    references: {
+      model: 'accomodations',
+      key: 'id' // key in Target model that we're referencing
+    },
+    onUpdate: 'CASCADE',
+    onDelete: 'SET NULL'
+  });
+  await queryInterface.addColumn('trips', 'userId', {
+    type: Sequelize.UUID,
+    references: {
+      model: 'users',
+      key: 'id' // key in Target model that we're referencing
+    },
+    onUpdate: 'CASCADE',
+    onDelete: 'SET NULL'
```

Comment:
This should also be cascade. If we delete a user their trip info is not as helpful since it's owner is not there anymore